ada_calibrate_camera/ada_calibrate_camera/publish_camera_extrinsics_node.py

```diff
@@ -1,4 +1,7 @@
-# !/usr/bin/env python3
+#!/usr/bin/env python3
```

Comment:
Inconsistent behavior since the other shebang lines were deleted, maybe because there's no space between # and !

Reply:
Before this PR, almost all code files had shebang lines. I removed the shebang line from all files but ROS2 nodes. Hence, it _seems like_ the only changes are deleting shebang lines, but in reality all ROS2 nodes (i.e., not helper files) still have shebang lines.

This ROS2 node had an incorrectly formatted shebang line (with the space), which I fixed. Although this is inconsistent with the other changes in the PR, it is consistent with the state of the codebase (shebang lines only in ROS2 node files).